src/main/resources/db/dev/V3_1__user_data.sql

```diff
@@ -197,7 +200,8 @@ VALUES ('ITAG_USER_ADM', 'ADMIN', 1, 'USER'),
 
 INSERT INTO DBA_USERS (username, account_status, profile)
 VALUES ('ITAG_USER', 'OPEN', 'TAG_GENERAL'),
-       ('ITAG_LADM', 'OPEN', 'TAG_ADMIN'),
+('ITAG_LADM', 'OPEN', 'TAG_ADMIN'),
+('HPA_USER', 'OPEN', 'TAG_GENERAL'),
 ('RESET_USER', 'OPEN', 'TAG_GENERAL'),
```

Comment:
I haven't added to sys.users table - wasn't sure if needed as the "create general user" endpoint doesn't add to this table